tests/checkers/unittest_symilar.py

```diff
@@ -497,3 +497,32 @@ def test_set_duplicate_lines_to_zero() -> None:
         similar.Run(["--duplicates=0", SIMILAR1, SIMILAR2])
     assert ex.value.code == 0
     assert output.getvalue() == ""
+
+
+@pytest.mark.parametrize("v", ["d"])
```

Comment:
This seems unnecessary?

Reply:
Yes, but with argparse we'll be able to do both again.